src/modules/wallet/components/overview/overview.js

```diff
@@ -80,7 +80,7 @@ const Overview = ({ isWalletRoute, history }) => {
         isBanned,
         pomHeight: pomHeights ? pomHeights[pomHeights.length - 1] : 0,
         readMore: () => {
-          const url = 'https://lisk.com/blog/development/lisk-staking-process';
+          const url = 'https://lisk.com/blog/posts/lisk-staking-process';
```

Comment:
can't we have this in a .env file  just as we do for the environment variable, PRODUCTION or DEBUG

Reply:
this was a category change.. in website they changed the category of research, development, posts, etc.. so this is just a one time thing.